gulp/tasks/webpack.js

```diff
@@ -3,11 +3,14 @@ import loadPlugins from 'gulp-load-plugins';
 import ManifestPlugin from 'webpack-manifest-plugin';
 import webpack from 'webpack';
 import WebpackDevServer from 'webpack-dev-server';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
 import path from 'path';
 import config from '../config.js';
+import Assets from './assets';
```

Comment:
This should have a more precise name... these are global assets that are expected to be loaded before main.js, so maybe 'GlobalAssets', or maybe 'UniversalLanguageSelectorDependencies' since that's all that's left that we need these globals for?